internal/models/reservation_model.go

```diff
@@ -123,6 +123,38 @@ type GCPReservation struct {
 	Detail *GCPDetail `db:"detail" json:"detail"`
 }
 
+type AzureDetail struct {
+	Location string `json:"location"`
+
+	// Instance name
+	Name string `json:"name"`
+
+	// GCP Machine type.
```

Comment:
```suggestion
	// Azure instance size.
```

Weird name tho, is this really how this is called on Azure? Looks like it: https://learn.microsoft.com/en-us/azure/virtual-machines/sizes

They call it sometimes Instances and sometimes Virtual Machines. So I like what you have, that is reasonable wording.

Reply:
yeah the terminology for Azure is not very well set, happy to change it if there is another suggestion, but in client I've seen InstanceSize, so I've used it.